modules/querying/pages/func/upload-udf-legacy.adoc

```diff
@@ -0,0 +1,83 @@
+= Uploading User-Defined Functions (Legacy)
+:sectnums:
+
+
+TigerGraph offers the ability to upload UDFs directly from GSQL.
+This allows authorized users to upload UDF files without having to access the server file system.
+
+For security reasons, this option is disabled by default and must be turned on by the TigerGraph Linux user.
+Once enabled, UDFs are no longer managed by `gadmin`.
+If your host machine is not completely secure, it is strongly recommended that you upload UDFs through `gadmin` instead of the steps on this page.
+
```

Comment:
@lennyjiaxichen This suggestion I leave up to you as this is something you know better than me, but I think it might be good to actually include a link _back_ to the recommended options here